roles/base/tasks/redhat_tasks.yml

```diff
@@ -22,3 +23,17 @@
 
 - name: install and start ntp
   service: name=ntpd state=started enabled=yes
+
+- name: enable NFS server
+  service:
+    name: nfs-server
+    state: started
+    enabled: yes
+  when: use_nfs
```

Comment:
In the unlikely event that they want both, this is more work with a 
significant drawback.

-Erik

On 27 Apr 2016, at 9:07, Madhav Puri wrote:

>> @@ -22,3 +23,17 @@
>>
>>  - name: install and start ntp
>>    service: name=ntpd state=started enabled=yes
>> +
>> +- name: enable NFS server
>> +  service:
>> +    name: nfs-server
>> +    state: started
>> +    enabled: yes
>> +  when: use_nfs
>
> We can have a more general variable like say `storage-type` with 
> possible values as `ceph`, `nfs` or `ceph,nfs`and so on which will let 
> us install either nfs or ceph or both based on user's deployment.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/180/files/9010d5bfb79e5fff3d0a25132af6edf21a833ff5#r61286978